docs/content/tutorial/step_04.ngdoc

```diff
@@ -259,7 +259,7 @@ After all the refactorings that took place, this is how our application looks fr
       phone-list.component.spec.js
       phone-list.module.js
       phone-list.template.html
-    app.css
+    app.js
```

Comment:
This change is invalid. We have `app.css` and `app.module.js` at the end of step 4.